list

```diff
@@ -710,12 +726,15 @@ lihipro.com
 lihivip.com
 liip.to
 lin.ee
+lingolink.ru
 link.ac
 link.gy
 link.infini.fr
+link.nn.ru
 link.tubi.tv
 linkbun.com
 linkd.in
+linkiro.co
```

Comment:
Any working examples for this one?

Reply:
Original link (https://linkiro.co/uU4f3g) still works for me, redirects with javascript:
```    <script>
    setTimeout(function() {
        window.location = 'https://github.com/PeterDaveHello/url-shorteners'
    }, 100);
    </script>
```
Archive: https://archive.is/0FQiy